lib/web_ui/test/engine/semantics/text_field_test.dart

```diff
@@ -34,6 +34,8 @@ void main() {
 }
 
 void testMain() {
+  ensureFlutterViewEmbedderInitialized();
```

Comment:
I'm a little confused by what initialization method needs to be called on each different type of test :S

Reply:
Some tests are testing something in isolation from the rest of the system. I didn't want to boil the ocean just yet. But you are right, we should consider a clean-up. We also have a lot of copy-paste of `setUpAll`, that could also be unified.